tests/vcd_disk.py

```diff
@@ -71,7 +71,7 @@ def test_008_update_disk(self):
         vdc = VDC(self.client, href=v.get('href'))
 
         
-        taskObj = vdc.update_disk(self.config['vcd']['disk'], "20", "DiskDisk1a", "20 MB Disk")
+        taskObj = vdc.update_disk(self.config['vcd']['disk'], "3072", "DiskUpdated", "3 GB Disk")
```

Comment:
Why is this change necessary?

Reply:
Wanted to test out with my own set of inputs.